tuist 0.1.0

pepicrft opened the pull request:
### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/bitrise-io/bitrise-steplib?pullRequest=3328) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/bitrise-io/bitrise-steplib?pullRequest=3328) before we can accept your contribution.<br/><hr/>**Pedro Piñera** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/bitrise-io/bitrise-steplib?pullRequest=3328) it.</sub>

koral-- commented on steps/tuist/0.1.0/step.yml:
```diff
@@ -0,0 +1,44 @@
+summary: |
+  Run Tuist from Bitrise's CI environment
+description: |
+  Run Tuist from Bitrise's CI environment
```
This text should be more descriptive. What is a tuist? What problem does it try to solve etc.

koral-- commented on steps/tuist/0.1.0/step.yml:
```diff
@@ -0,0 +1,44 @@
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
```
Curl is used not those listed ones.

pepicrft commented:
Thanks @koral-- for the review. I addressed your comments and pushed a commit with the fixes. Please, let me know if there's anything else missing.

godrei commented on steps/tuist/0.1.0/step.yml:
```diff
@@ -0,0 +1,46 @@
+title: tuist
```
Based on the Step Title conventions: `Run tuist` (or something like this) would be better: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title

godrei commented on steps/tuist/0.1.0/step.yml:
```diff
@@ -0,0 +1,46 @@
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Based on the Step's source code only `curl` is used as an external dependency, pls do not list unused dependencies to speed up the step run.

godrei reviewed (changes requested):
Hi @pepicrft ,
thanks for the changes, I spotted 2 more issues, please have a look at them.

Based on the CLA assistant, it seems the 2 commits on this PR were done by different GH users (`pepicrft` and `Pedro Piñera`). Please sign the CLA with both users or[ re-share the step](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/sharing-steps-with-all-bitrise-users.html#fixing-issues-in-a-steplib-pull-request).

pepicrft commented:
Thanks for the review @godrei. I addressed your feedback and the CLA agreement seems to be ok now. Please, let me know if there's anything missing on my end.

godrei commented:
Hi @pepicrft ,
thanks for the updates, everything looks good to me.
